mobilecoind/clients/python/mailchimp/allocate_mobilecoins.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018-2020 MobileCoin Inc.
+
+# used to allocation TestNet mobilecoins to users who sign up for our mailing list
+
+import argparse
+from mailchimp3 import MailChimp
+import time
+
+import os,sys
+sys.path.insert(1, os.path.realpath(os.path.join(os.path.pardir, "lib")))
+import mobilecoin
+
+TX_RECEIPT_CHECK_INTERVAL_SECONDS = 2
+
+def wait_for_monitor(monitor_id):
+    (monitor_is_behind, next_block, remote_count, blocks_per_second) = mobilecoind.wait_for_monitor(monitor_id)
+    if monitor_is_behind:
+        print("#\n# waiting for the sender's monitor to process {} blocks".format(remote_count - next_block))
+        while monitor_is_behind:
+            blocks_remaining = (remote_count - next_block)
+            if blocks_per_second > 0:
+                time_remaining_seconds = blocks_remaining / blocks_per_second
+                print("#    {} blocks remain ({} seconds)".format(blocks_remaining, round(time_remaining_seconds, 1)))
+            else:
+                print("#    {} blocks remain (? seconds)".format(blocks_remaining))
+            (monitor_is_behind, next_block, remote_count, blocks_per_second) = mobilecoind.wait_for_monitor(monitor_id)
+        print("# sender's  monitor has processed all {} blocks\n#".format(local_count))
+    return remote_count
+
+# generates a new master key, allocates funds, stores data at Mailchimp and triggers the welcome email
```

Comment:
Prefer [PEP-8 styling](https://www.python.org/dev/peps/pep-0008/#documentation-strings) for doc strings